plugins/in_node_exporter_metrics/ne.c

```diff
@@ -701,6 +720,26 @@ static int in_ne_init(struct flb_input_instance *in,
                     }
                     ne_systemd_init(ctx);
                 }
+                else if (strncmp(entry->str, "processes", 9) == 0) {
+                    if (ctx->processes_scrape_interval == 0) {
+                        flb_plg_debug(ctx->ins, "enabled metrics %s", entry->str);
```

Comment:
it seems process collector is disabled but the message says enabled

Reply:
This message is used when enabling processes metrics. Other messages for enabling metrics are also displayed in enabled circumstances.